tests/plugins/test_koji_upload.py

```diff
@@ -1179,36 +1023,15 @@ def test_koji_promote_pullspec(self, tmpdir, os_env, primary, unique, invalid):
         else:
             assert pullspec.endswith('-timestamp')
 
-    def test_koji_promote_without_build_info(self, tmpdir, os_env):
-
-        class LegacyCGImport(MockedClientSession):
-
-            def CGImport(self, *args, **kwargs):
-                super(LegacyCGImport, self).CGImport(*args, **kwargs)
-                return
-
-        session = LegacyCGImport('')
-        name = 'ns/name'
-        version = '1.0'
-        release = '1'
-        tasker, workflow = mock_environment(tmpdir,
-                                            session=session,
-                                            name=name,
-                                            version=version,
-                                            release=release)
-        runner = create_runner(tasker, workflow)
-        runner.run()
-
-        assert runner.plugins_results[KojiPromotePlugin.key] is None
-
     @pytest.mark.parametrize('expect_result', [
         'empty_config',
         'no_help_file',
         'skip',
         'pass',
         'unknown_status'
     ])
-    def test_koji_promote_add_help(self, tmpdir, os_env, expect_result):
+    def test_koji_upload_add_help(self, tmpdir, os_env, expect_result):
```

Comment:
This test is concerned with the 'build' key which koji_upload will not create, so it can be removed.